core/entities/models/JsonModelSchema.php

```diff
@@ -112,6 +123,45 @@ public function getModelSchemaForRelations(array $relations_on_model, array $sch
                 : EEH_Inflector::pluralize_and_lower($model_name);
             $schema['properties'][ $model_name_for_schema ] = $relation->getSchema();
             $schema['properties'][ $model_name_for_schema ]['relation_model'] = $model_name;
+
+            // links schema
+            $links_key = 'https://api.eventespresso.com/' . strtolower($model_name);
+            $schema['properties']['_links']['properties'][ $links_key ] = array(
```

Comment:
Your suggestion to implement the actual relation links was a good one so I went ahead and did this here.